test/e2e/operator_test.go

```diff
@@ -137,30 +145,49 @@ func TestDefaultContour(t *testing.T) {
 
 	// Create a sample workload for e2e testing.
 	appName := fmt.Sprintf("%s-%s", testAppName, testName)
-	if err := newDeployment(ctx, kclient, appName, specNs, testAppImage, testAppReplicas); err != nil {
-		t.Fatalf("failed to create deployment %s/%s: %v", specNs, appName, err)
+	if err := newDeployment(ctx, kclient, appName, cfg.SpecNs, testAppImage, testAppReplicas); err != nil {
```

Comment:
If I'm reading the test right, this is still using `projectcontour` as the namespace to run the deployment etc. in b/c the `cfg.SpecNs` value hasn't been changed from `specNs`, is that intended for the `TestDefaultContour` test?

Reply:
@sunjayBhatia that's correct.